.settings/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.gnumcueclipse.debug.gdbjtag.openocd.prefs
+*.gnumcueclipse.managedbuild.cross.arm.prefs
+*.settings.xml
```

Comment:
Can all of the gitignores be consolidated to just the root `.gitignore` file? Adding a new top level folder isn't ideal.

Reply:
This gitignore file is comes from the used IDE Eclipse. I think it is not necessary and we can remove it in the repo.